src/clients/migration_proposals.rb

```diff
@@ -1,3 +1,23 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2015 SUSE GmbH, All Rights Reserved.
```

Comment:
Should it be `SUSE LLC`? I'm not sure, but in `registration` and `journal` there is `SUSE LLC`.

Who is actually the correct copyright holder now?